packages/react-app/.babelrc

```diff
@@ -8,10 +8,11 @@
         "corejs": 3
       }
     ],
-    "@babel/preset-react"
+    "@babel/preset-react",
+    "@babel/preset-typescript"
   ],
   "plugins": [
-    ["styled-components", { "displayName": true }],
+    ["babel-plugin-styled-components", { "displayName": true }],
     "@babel/plugin-transform-react-jsx-source",
     "@babel/plugin-proposal-class-properties"
```

Comment:
Maybe use `["@babel/plugin-proposal-class-properties", { "loose": true }]` or remove completely, because the react preset have it already